tests/system/action/meeting/test_settings.py

```diff
@@ -34,4 +34,7 @@ def test_html_field(self) -> None:
         response = self.request("meeting.update", {"id": 1, "welcome_text": "<iframe>"})
         self.assert_status_code(response, 200)
         meeting = self.get_model("meeting/1")
-        self.assertEqual(meeting["welcome_text"], "&lt;iframe&gt;")
```

Comment:
Please find a tag which is still forbidden and add a test to check that forbidden tags are escaped correctly.

Reply:
I used the "script" tag to test the escaping. Add this new test and rename the tests.